projects/exui-common-lib/src/lib/services/case-sharing-state/case-sharing-state.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { distinctUntilChanged } from 'rxjs/operators';
+import { SharedCase } from '../../models/case-share.module';
+import { UserDetails } from '../../models/user-details.module';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CaseSharingStateService {
+
+  private caseState: SharedCase[] = [];
+  private readonly subject = new BehaviorSubject<SharedCase[]>(this.caseState);
+
+  public get state() {
+    return this.subject.asObservable().pipe(
+      distinctUntilChanged()
+    );
+  }
+
+  constructor() { }
+
+  public setCases(orgId: string, cases: SharedCase[]): void {
+    // TODO: check organisation policy against orgId
+    parseInt(orgId, 10); // do something with orgId for now to stop linting errors
```

Comment:
Not understanding why this is required? If orgId is meant to be an int then shouldn't it be typed as such?

Reply:
orgId isn't being used, can you not refactor the method (can probably use your ide for this) to remove it?